docs/_docs/snapshots/snapshots.adoc

```diff
@@ -95,6 +95,16 @@ increases the total amount of time for taking a snapshot. However, this keeps th
 
 See the link:perf-and-troubleshooting/thread-pools-tuning[Ignite Snapshot Execution Pool,window=_blank] page for more details.
 
+=== Distributed properties
+
+The distributed properties listed in the table below allow you to configure snapshots at runtime:
```

Comment:
Please, reflect in this documentation changes made in the source code.

Reply:
Done.